From cc24c84d35274aead5afe574c87ecd6368aa5e2a Mon Sep 17 00:00:00 2001
From: Kohei Yoshida <kohei.yoshida@gmail.com>
Date: Tue, 2 Nov 2021 18:24:30 -0400
Subject: [PATCH] More unused parameters.

I personally cannot reproduce the warnings. But The android x86 build
of LibreOffice generates the following build warnings:

In file included from /home/tdf/lode/jenkins/workspace/android_x86/svl/source/misc/gridprinter.cxx:16:
In file included from /home/tdf/lode/jenkins/workspace/android_x86/workdir/UnpackedTarball/mdds/include/mdds/multi_type_matrix.hpp:37:
In file included from /home/tdf/lode/jenkins/workspace/android_x86/workdir/UnpackedTarball/mdds/include/mdds/multi_type_vector.hpp:31:
In file included from /home/tdf/lode/jenkins/workspace/android_x86/workdir/UnpackedTarball/mdds/include/mdds/./multi_type_vector/soa/main.hpp:35:
/home/tdf/lode/jenkins/workspace/android_x86/workdir/UnpackedTarball/mdds/include/mdds/./multi_type_vector/soa/./block_util.hpp:49:27: error: unused parameter 'block_store' [-Werror,-Wunused-parameter]
    void operator()(Blks& block_store, int64_t start_block_index, int64_t delta) const
                          ^
/home/tdf/lode/jenkins/workspace/android_x86/workdir/UnpackedTarball/mdds/include/mdds/./multi_type_vector/soa/./block_util.hpp:49:48: error: unused parameter 'start_block_index' [-Werror,-Wunused-parameter]
    void operator()(Blks& block_store, int64_t start_block_index, int64_t delta) const
                                               ^
/home/tdf/lode/jenkins/workspace/android_x86/workdir/UnpackedTarball/mdds/include/mdds/./multi_type_vector/soa/./block_util.hpp:49:75: error: unused parameter 'delta' [-Werror,-Wunused-parameter]
    void operator()(Blks& block_store, int64_t start_block_index, int64_t delta) const
                                                                          ^
3 errors generated.
---
 include/mdds/multi_type_vector/soa/block_util.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/mdds/multi_type_vector/soa/block_util.hpp b/include/mdds/multi_type_vector/soa/block_util.hpp
index d10ee67..73ba27c 100644
--- a/include/mdds/multi_type_vector/soa/block_util.hpp
+++ b/include/mdds/multi_type_vector/soa/block_util.hpp
@@ -46,7 +46,7 @@ namespace detail {
 template<typename Blks, lu_factor_t F>
 struct adjust_block_positions
 {
-    void operator()(Blks& block_store, int64_t start_block_index, int64_t delta) const
+    void operator()(Blks& /*block_store*/, int64_t /*start_block_index*/, int64_t /*delta*/) const
     {
         static_assert(invalid_static_int<F>, "The loop-unrolling factor must be one of 0, 4, 8, 16, or 32.");
     }
-- 
2.25.1

